drivers/spi/spi_mcux_flexcomm.c

```diff
@@ -1,12 +1,13 @@
 /*
  * Copyright (c) 2016, Freescale Semiconductor, Inc.
- * Copyright (c) 2017,2019, NXP
+ * Copyright 2017,2019,2025, NXP
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 #define DT_DRV_COMPAT nxp_lpc_spi
 
+
```

Comment:
very minor nit, but there are some extra empty lines added here and below that can be removed